ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-vrf-address-families.md

```diff
@@ -136,11 +138,20 @@ router bgp 65001
          neighbor 1.2.3.4 additional-paths send any
          network 2.3.4.0/24 route-map BARFOO
          no bgp redistribute-internal
-         redistribute connected rcf VRF_AFIPV4_RCF_CONNECTED_1()
-         redistribute ospf match external
-         redistribute ospf match nssa-external 1
-         redistribute ospfv3 match internal
-         redistribute static route-map VRF_AFIPV4_RM_STATIC_1
+         redistribute attached-host route-map VRF_AFIPV4_RM_HOST
+         redistribute bgp leaked route-map VRF_AFIPV4_RM_BGP
+         redistribute connected include leaked rcf VRF_AFIPV4_RCF_CONNECTED_1()
+         redistribute dynamic route-map VRF_AFIPV4_RM_DYNAMIC
+         redistribute isis level-1 include leaked rcf VRF_AFIPV4_RCF_ISIS()
+         redistribute ospf include leaked route-map VRF_AFIPV4_RM_OSPF
+         redistribute ospf match external include leaked route-map VRF_AFIPV4_RM_OSPF
+         redistribute ospf match nssa-external 1 include leaked route-map VRF_AFIPV4_RM_OSPF
+         redistribute ospfv3 match internal include leaked route-map VRF_AFIPV4_RM_OSPF
+         redistribute ospfv3 match external include leaked route-map VRF_AFIPV4_RM_OSPF
+         redistribute ospfv3 match nssa-external 2 route-map VRF_AFIPV4_RM_OSPF
+         redistribute rip route-map VRF_AFIPV4_RM_RIP
+         redistribute static include leaked route-map VRF_AFIPV4_RM_STATIC_1
+         redistribute user rcf VRF_AFIPV4_RCF_USER()
```

Comment:
Order mismatch **(user, ospf, ospfv3)** -
```
   address-family ipv4
      redistribute attached-host route-map VRF_AFIPV4_RM_HOST
      redistribute bgp leaked route-map VRF_AFIPV4_RM_BGP
      redistribute connected include leaked rcf VRF_AFIPV4_RCF_CONNECTED_1()
      redistribute dynamic route-map VRF_AFIPV4_RM_DYNAMIC
      redistribute user rcf VRF_AFIPV4_RCF_USER()
      redistribute isis level-1 include leaked rcf VRF_AFIPV4_RCF_ISIS()
      redistribute ospf include leaked route-map VRF_AFIPV4_RM_OSPF
      redistribute ospfv3 match internal include leaked route-map VRF_AFIPV4_RM_OSPF
      redistribute ospfv3 match external include leaked route-map VRF_AFIPV4_RM_OSPF
      redistribute ospfv3 match nssa-external 2 route-map VRF_AFIPV4_RM_OSPF
      redistribute ospf match external include leaked route-map VRF_AFIPV4_RM_OSPF
      redistribute ospf match nssa-external 1 include leaked route-map VRF_AFIPV4_RM_OSPF
      redistribute rip route-map VRF_AFIPV4_RM_RIP
```

Reply:
Please check the order at other places as well

Reply:
updated